### PR TITLE
Add snippet for public const, enum, struct & type

### PR DESCRIPTION
**Pull request by dhonx**

### CODE REVIEW

File: snippets/snippets.json [hunk 1]
```diff
@@ -191,12 +191,36 @@
...
+	"snippet.public.const": {
+		"body": ["pub const ($1)"],
+		"description": "Code snippet for public constant 'pub const'",
+		"prefix": "pubc",
+		"scope": "v,vlang"
+	},
+	"snippet.public.enum": {
+		"body": ["pub enum ${1:name} {", "\t$0", "}"],
+		"description": "Code snippet for public enum 'pub enum'",
+		"prefix": "pube",
```
Comment (Alhadis) [inline snippets/snippets.json]: How about `pe`, `pc`, `ps`, etc? It's shorter and easy enough to remember:

`pc` = <b><ins>p</ins></b>ublic <b><ins>c</ins></b>onst
`pe` = <b><ins>p</ins></b>ublic <b><ins>e</ins></b>num
`ps` = <b><ins>p</ins></b>ublic <b><ins>s</ins></b>truct
… etc

Comment (dhonx) [reply snippets/snippets.json]: good idea !!!

Comment (dhonx) [reply snippets/snippets.json]: so `pf` for `pu fn`

Comment (Alhadis) [reply snippets/snippets.json]: I think so. 👍 The current prefix `pub` is a little ambiguous.

**Pull request closed without merging**